src/Core/Framework/Adapter/Twig/TwigEnvironment.php

```diff
@@ -23,6 +23,7 @@ public function __construct(LoaderInterface $loader, array $options = [])
     {
         // There is no Symfony configuration yet to toggle this feature
         $options['use_yield'] = true;
+        $options['debug'] = false;
```

Comment:
is this needed? That should be configurable in the environment and not hard coded here right?

Reply:
It can be removed; I tried to use it for debugging ...